### PR TITLE
25 check pkgtitle for python

### PR DESCRIPTION
**Pull request by jwalton3141**
Need `intern = TRUE` in call to `system` to return python package "title"

### CODE REVIEW

File: R/check_pkgtitle.R [hunk 1]
```diff
@@ -11,7 +11,8 @@ check_pkgtitle = function() {
     pkg_title = packageDescription(r_pkg)$Title
   } else if (language == "python") {
     python_pkg = get_python_pkg_name()
-    pkg_title = system(glue::glue("pip show {python_pkg} | sed -n -e 's/Summary: //p'"))
+    pkg_title = system(glue::glue("pip show {python_pkg} | sed -n -e 's/Summary: //p'"),
```
Comment (csgillespie) [inline R/check_pkgtitle.R:14]: Can you change this to `system2`. Something like:
```
system2("pip", args = c("show", {python_pkg}, "|", "sed", "-n", "-e", ....), stdout = TRUE)
```

**Review by csgillespie — CHANGES_REQUESTED** [review]

**Review by jwalton3141 — COMMENTED** [review]
Added nolint here because of the curly braces. Not sure whether this was the "proper" solution.

File: R/check_pkgtitle.R [hunk 2]
```diff
@@ -11,7 +11,12 @@ check_pkgtitle = function() {
     pkg_title = packageDescription(r_pkg)$Title
   } else if (language == "python") {
     python_pkg = get_python_pkg_name()
-    pkg_title = system2("pip", args = c("show", {python_pkg}, "|", "sed", "-n", "-e", "'s/Summary: //p'"), stdout = TRUE)
+    # nolint start
+    # the curly braces in {python_pkg} trigger the lintr
+    pkg_title = system2("pip", args = c("show", {python_pkg},
```
Comment (csgillespie) [inline R/check_pkgtitle.R:16]: You don't need curly braces here (I think) @jwalton3141

Comment (jwalton3141) [reply R/check_pkgtitle.R]: yep, ta. noted and dropped in next commit

**Review by csgillespie — APPROVED** [review]

**Pull request merged**